server/src/main/java/org/opensearch/search/profile/aggregation/ConcurrentAggregationProfiler.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
Please keep only OpenSearch licensing (this is new file, no attribution to Elasticsearch)

Reply:
Good call, thanks. It has been fixed now.